### DISCUSSION TITLE
Tracking stats on which words are difficult?

### DISCUSSION THREAD

**Discussion by astrowonk** [octokatherine/word-master#42] [Ideas]
Hello, I made [thetwitterbot](https://twitter.com/thewordlebot)(not so much a bot yet as me pasting in the results...) where I am algorithmically generating guesses, and there are 44 words the code can't solve from the complete ~12000 length wordle list (depends on various factors, but that's the lowest I've gotten it). I'd be curious to share notes if these are the same words human players also struggle to solve?

**Comment by octokatherine** [comment]
Hey, right now there's not an easy way to implement that since I don't store any user data, it's just a static site with streak values saved in local storage

**Reply by astrowonk** [reply]
Ah, that makes sense. I ... probably could have figured that out if I looked at the code, ha.

I've been reluctant to look at or use the actual ~2000 word wordle target list so it's possible some of these harder words aren't even in there.
